guides/common/modules/con_remote-execution-workflow.adoc

```diff
@@ -1,7 +1,11 @@
 [id="remote-execution-workflow_{context}"]
 = Remote Execution Workflow
 
-When you run a remote job on hosts, for every host, {Project} performs the following actions to find a remote execution {SmartProxy} to use.
+{Project} prefers remote execution on host's registered_through proxy.
```

Comment:
> what is "registered_through"? Please reword to make it less technical.

The original idea from a reporter was to keep it like this. But, we are Okay to reword it.

> describe what REX is

We have used the phrase **Remote Execution** instead of REX.

> how SmartProxies play their part to run jobs on managed hosts.

Will again reword it. :+1: